src/main/java/com/github/fishio/PlayerFish.java

```diff
@@ -271,7 +263,7 @@ public void setRightPressed(boolean rightPressed) {
 	 * @return The rate at which the PlayerFish grows.
 	 */
 	public double getGrowthSpeed() {
-		return GROWTH_SPEED;
+		return settings.getInteger("GROWTH_SPEED");
```

Comment:
Then change the method to return an int, and make sure that all calculations using it are still double precision.

<organisation